vertical-pod-autoscaler/README.md

```diff
@@ -298,9 +304,9 @@ You can then choose which recommender to use by setting `recommenders` inside th
 
 # Known limitations
 
-* Updating running pods is an experimental feature of VPA. Whenever VPA updates
-  the pod resources, the pod is recreated, which causes all running containers to
-  be restarted. The pod may be recreated on a different node.
+* Whenever VPA updates the pod resources, the pod is recreated, which causes all
+  running containers to be restarted. The pod may be recreated on a different
```

Comment:
To be precise the containers are not restarted, they're also created anew. This might be an important distinction if someone attaches any external resources to these containers (e.g. persistent disks).